packages/web-components/src/select/select.styles.ts

```diff
@@ -0,0 +1,169 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  // Design tokens
+  borderRadiusMedium,
+  colorCompoundBrandStroke,
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  colorNeutralForegroundDisabled,
+  colorNeutralStroke1,
+  colorNeutralStrokeAccessible,
+  colorNeutralStrokeDisabled,
+  colorTransparentBackground,
+  curveAccelerateMid,
+  curveDecelerateMid,
+  durationNormal,
+  durationUltraFast,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontWeightRegular,
+  lineHeightBase200,
+  lineHeightBase300,
+  lineHeightBase400,
+  spacingHorizontalMNudge,
+  spacingHorizontalXXS,
+} from '../theme/design-tokens.js';
+
+/** Text styles
+ * @public
+ */
+export const styles = css`
+  ${display('block')} :host {
+    font-family: ${fontFamilyBase};
```

Comment:
With the font family set here, do you need to redeclare for label and select? Won't they inherit this by default?